src/include/alloc.h

```diff
@@ -37,13 +37,28 @@ static ncclResult_t ncclCalloc(T** ptr, size_t nelem) {
   return ncclSuccess;
 }
 
+struct __attribute__ ((aligned(64))) allocationTracker {
+  uint64_t totalAlloc;
```

Comment:
Can we just use a union with one field being 64 chars?

Reply:
done